src/js/rtp-stats.js

```diff
@@ -62,42 +61,32 @@ export function extractMediaStatsFromStats(timestamp, stats, streamType) {
 
                 }
             // Standardized report for input stream stats
-            } else if (statsReport.type === 'inbound-rtp' && !statsReport.isRemote && (streamType === 'audio_input' || streamType === 'video_input')) {
```

Comment:
Do we need the above legacy stats extraction logic if we are moving to standardized stats?
We support only the latest 3 browser versions so we should be good to remove lines 17 - 62?

Reply:
Yes, you are right. I was initially considering enabling the standardized path first, then deprecating and cleaning up  the legacy code path as a follow up task, but it doesn't seem to provide any benefits. In the next revision, I will simply remove them. Thank you